tools/internal_ci/macos/grpc_cfstream.cfg

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 gRPC authors.
```

Comment:
"2019 The gRPC Authors" + no dot.

https://github.com/grpc/grpc/blob/7ef8fc826c52b0c4abeaead633d464197fc0bdf8/doc/python/sphinx/conf.py#L1

Reply:
Sorry, I missed this comment earlier. Created PR #18042 to fix the copyright string.